Sources/ArcGISToolkit/Components/AR/TableTopSceneView.swift

```diff
@@ -117,11 +118,26 @@ public struct TableTopSceneView: View {
                     }
                 }
                 .onAppear {
-                    arViewProxy.session?.run(configuration)
+                    arViewProxy.session.run(configuration)
                 }
                 .onDisappear {
-                    arViewProxy.session?.pause()
+                    arViewProxy.session.pause()
+                }
+            
+            ARCoachingOverlay(goal: .horizontalPlane)
```

Comment:
Instead of messing with the opacity, I think I would just put this in an if block:

```swift
if !coachingOverlayIsHidden { 
    ...
}
```